pyfr/solvers/euler/kernels/entropyfilter.mako

```diff
@@ -90,32 +95,36 @@
               vdm='in broadcast fpdtype_t[${str(nefpts)}][${str(nupts)}]'
               invvdm='in broadcast fpdtype_t[${str(nupts)}][${str(nupts)}]'
               m0='in broadcast fpdtype_t[${str(nfpts)}][${str(nupts)}]'>
-    fpdtype_t dmin, pmin, emin;
+
+    // X = rho*(s-s0)
+    fpdtype_t dmin, pmin, emin, Xmin;
```

Comment:
Is emin still used anymore?

Reply:
emin is used at the end of the filtering operation to store the new local minimum specific entropy through the get_minima expansion, but not in the interim.